tests/test_errors.py

```diff
@@ -552,10 +588,10 @@ def finalize(self, cfg: Any) -> None:
     pytest.param(
         Expected(
             create=lambda: OmegaConf.structured(SubscriptedDict),
-            op=lambda cfg: cfg.__setitem__("dict", 1),
+            op=lambda cfg: cfg.__setitem__("dict_str", 1),
```

Comment:
why change that?

Reply:
Previously in the `tests/__init__.py` file we had:
```python
@dataclass
class SubscriptedDict:
    dict: Dict[str, int] = field(default_factory=lambda: {"foo": 4})
```

I changed this `SubscriptedDict` class to look like this:
```python
@dataclass
class SubscriptedDict:
    dict_str: Dict[str, int] = field(default_factory=lambda: {"foo": 4})
    dict_enum: Dict[Color, int] = field(default_factory=lambda: {Color.RED: 4})
    dict_int: Dict[int, int] = field(default_factory=lambda: {123: 4})
    dict_float: Dict[float, int] = field(default_factory=lambda: {123.45: 4})
    dict_bool: Dict[bool, int] = field(default_factory=lambda: {True: 4, False: 5})
```
So the `dict` attribute got renamed to `dict_str`.